testgrid/tgrun/pkg/scheduler/scheduler.go

```diff
@@ -43,6 +43,12 @@ func Run(schedulerOptions types.SchedulerOptions) error {
 
 	for _, instance := range kurlPlans {
 		testSpec := instance.InstallerSpec
+		// append sonobuoy for conformance testing
```

Comment:
maybe we should only run sonobouy on some of the installs, instead of all of them?